### DISCUSSION TITLE
Assigned Issues List

### DISCUSSION THREAD

**Discussion by lizable** [lablup/contribution-academy-2021#85] [General]
Please add a comment about the issue link that you want to solve.
> 🚨  NOTE:  If someone already added the issue that they want to solve, please choose another issue.

Please check the example comment below
⬇️ ⬇️⬇️

**Comment by lizable** [comment]
@lizable I'll handle webui#1068(https://github.com/lablup/backend.ai-webui/issues/1068)

**Comment by euni00** [comment]
@lizable I'll handle webui#1110(lablup/backend.ai-webui#1110)

**Comment by youngjun0627** [comment]
@lizable I'll handle discussion#76([lablup/contribution-2021-academy-2021#76(discussion)](https://github.com/lablup/contribution-academy-2021/discussions/76))[webui] and issue#227([lablup/backend.ai#227](https://github.com/lablup/backend.ai/issues/227))[manager, agent, client]

**Comment by TeusEE** [comment]
@lizable storage_proxy#11(https://github.com/lablup/backend.ai-storage-proxy/issues/11#issue-743489261)

**Reply by TeusEE** [reply]
@lizable is this issue has been solved already?

in the issue, mention that he need to move_tree function.

latest vfs/__ini__.py, there is move_tree.

is this issue need to be ended?

**Reply by TeusEE** [reply]
https://github.com/lablup/backend.ai-storage-proxy/pull/27

**Reply by lizable** [reply]
@TeusEE Please check whether this issue has been solved already. Just simply do test by creating new folder inside any vfolder, and try to rename it. If it works, then this issue doesn't need to be solved.

**Comment by 1hk-star** [comment]
@lizable l'll handle webui#1094(https://github.com/lablup/backend.ai-webui/issues/1094)

**Comment by TeusEE** [comment]
@lizable i'll take this issue
https://github.com/lablup/backend.ai/issues/67

**Comment by eunjungchoi** [comment]
@lizable i'll take this issue
[lablup/backend.ai#283
](https://github.com/lablup/backend.ai/issues/283)

**Comment by yujung7768903** [comment]
@lizable  I'll handle webui#1123(lablup/backend.ai-webui#1123)

**Comment by TeusEE** [comment]
@lizable  i'll take this issue
https://github.com/lablup/backend.ai-webui/issues/1131

**Reply by lizable** [reply]
I'll handle this issue. Could you do [lablup/backend.ai-webui#1074](https://github.com/lablup/backend.ai-webui/issues/1074) instead of this issue?
